tests/testinterop.nim

```diff
@@ -2,6 +2,8 @@ import helpers, commoninterop
 import ../libp2p
 import ../libp2p/crypto/crypto, ../libp2p/protocols/connectivity/relay/relay
 
+{.used.}
```

Comment:
other `testXXX` files have `{.used.}` at the top (first line). can we keep the same convention?